core/src/main/scala/io/qbeast/core/transform/LinearTransformation.scala

```diff
@@ -38,17 +43,19 @@ case class LinearTransformation(minNumber: Any, maxNumber: Any, orderedDataType:
 
   private val scale: Double = {
     val mx = maxNumber.toDouble
-    require(mx > mn, "Range cannot be not null, and max must be > min")
-    1.0 / (mx - mn)
+    require(mx >= mn, "Range cannot be not null, and max must be >= min")
+    if (mx == mn) 0.0 else 1.0 / (mx - mn)
```

Comment:
Please just check this and let me know if it's correct :)

In case **min and max are equal** (nulls or not), `scale` should be `0.0`

Reply:
I think that's correct now. Sorry for the late response!